extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/providers/DsChannelTypeProvider.java

```diff
@@ -189,6 +194,17 @@ public static boolean isOutputChannel(String channelTypeID) {
         return supportedOutputChannelTypes.contains(channelTypeID);
     }
 
+    @Reference
```

Comment:
In the XML it was declared 1..1 / **DYNAMIC**. The implementation actually can handle it being `null`, but as it is mandatory (1..1) anyway, I agree it does make more sense to declare it statically. Therefore: all good 😃